organization/src/integrationTest/kotlin/com/cosmotech/organization/service/OrganizationServiceIntegrationTest.kt

```diff
@@ -1283,1447 +1283,1502 @@ class OrganizationServiceIntegrationTest : CsmRedisTestBase() {
-          }
-          val otherUserACL = OrganizationAccessControl(id = OTHER_TEST_USER_ID, role = ROLE_VIEWER)
-          assertNotNull(
-              organizationApiService.addOrganizationAccessControl(
-                  organizationRegistered.id!!, otherUserACL))
-
-          val otherUserACLRetrieved =
-              organizationApiService.getOrganizationAccessControl(
-                  organizationRegistered.id!!, OTHER_TEST_USER_ID)
-          assertEquals(otherUserACL, otherUserACLRetrieved)
-        }
+        assertEquals(otherUserACL, otherUserACLRetrieved)
       }
+    }
 
-      @Test
-      fun `addOrganizationAccessControl with owned organization (ROLE_NONE)`() {
-        assertThrows<CsmClientException> {
-          val name = "o-connector-test-1"
-          val organizationRegistered =
-              organizationApiService.registerOrganization(createTestOrganization(name))
+    @Test
+    fun `addOrganizationAccessControl with owned organization (ROLE_NONE)`() {
+      assertThrows<CsmClientException> {
+        val name = "o-connector-test-1"
+        val organizationRegistered =
+            organizationApiService.registerOrganization(createTestOrganization(name))
 
-          val otherUserACL = OrganizationAccessControl(id = OTHER_TEST_USER_ID, role = ROLE_NONE)
-          organizationApiService.addOrganizationAccessControl(
-              organizationRegistered.id!!, otherUserACL)
-        }
+        val otherUserACL = OrganizationAccessControl(id = OTHER_TEST_USER_ID, role = ROLE_NONE)
+        organizationApiService.addOrganizationAccessControl(
+            organizationRegistered.id!!, otherUserACL)
       }
+    }
 
-      @Test
-      fun `addOrganizationAccessControl with not owned organization and PERMISSION_WRITE_SECURITY`() {
-        assertDoesNotThrow {
-          val name = "o-connector-test-1"
-          runAsDifferentOrganizationUser()
-          val organizationRegistered =
-              organizationApiService.registerOrganization(
-                  createTestOrganizationWithSimpleSecurity(
-                      name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_ADMIN))
-          runAsPlatformAdmin()
-          assertThrows<CsmResourceNotFoundException> {
-            organizationApiService.getOrganizationAccessControl(
-                organizationRegistered.id!!, TEST_USER_ID)
-          }
-          val otherUserACL = OrganizationAccessControl(id = TEST_USER_ID, role = ROLE_VIEWER)
-          assertNotNull(
-              organizationApiService.addOrganizationAccessControl(
-                  organizationRegistered.id!!, otherUserACL))
-
-          val otherUserACLRetrieved =
-              organizationApiService.getOrganizationAccessControl(
-                  organizationRegistered.id!!, TEST_USER_ID)
-          assertEquals(otherUserACL, otherUserACLRetrieved)
+    @Test
+    fun `addOrganizationAccessControl with not owned organization and PERMISSION_WRITE_SECURITY`() {
+      assertDoesNotThrow {
```

Comment:
assertDoesNotThrow should be specific else it's the same as not having it

Reply:
same comment as above